src/Assets.jl

```diff
@@ -14,22 +30,27 @@ Cash(S::Symbol) = Cash{S,unit(S)}()
 Cash(::Currency{S}) where {S} = Cash(S)
 
 unit(::Cash{C,N}) where {C,N} = N
-code(c::Cash) = code(currency(c))
-name(c::Cash) = name(currency(c))
+currency(::Cash{C}) where {C} = C
+code(c::Cash)   = code(currency(c))
+name(c::Cash)   = name(currency(c))
+symbol(c::Cash) = symbol(currency(c))
 
 Base.show(io::IO, ::Cash{C}) where {C} = print(io, string(C))
 
 function Position{Cash{C,N}}(a) where {C,N}
     T = FixedDecimal{Int,N}
     Position{Cash{C,N},T}(T(a))
 end
+Position(c::C, v) where {C<:Cash} = Position{typeof(c)}(v)
```

Comment:
I don't think we want this constructor. It is already removed from Instruments.jl so to get the test to pass, I'd suggest just removing the test that uses this obsolete constructor.

Reply:
If you really want this constructor (which is a remnant of when `Position` had an `instrument` property, we can put the general one in `Instruments.jl` and it would cover `Cash` as well. In any case, I think it doesn't belong here.